draft-ietf-rats-corim.md

```diff
@@ -255,8 +256,20 @@ convention - e.g., {{Section 4.4.1.5 of -psa-token}}).
 ~~~
 
 # CoRIM
+Concise Reference Integrity Manifests (CoRIM) contain tags that describe the
```

Comment:
The sentence can simply use 'CoRIM' since the expanded form is in the heading.
Suggest singular form: "A CoRIM contains..."